sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md

```diff
@@ -1,10 +1,6 @@
 # Release History
 
-## 5.8.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 5.7.6 (2023-03-07)
```

Comment:
Sorry for not doing this already, but we should add a note about the OpenTelemetry updates.

Reply:
Also, it might be worth bumping to a new minor version for that.

Reply:
Got some verbiage to throw me and I'll add it in?